book/src/concepts/component.md

```diff
@@ -16,19 +16,31 @@ In this example, the bottle is the entity, and the properties are components.
 
 Creating a component is easy.
 
-You declare a struct:
+You declare the relevant structs:
 
 ```rust,ignore
-pub struct MyComponent {
-    some_property: String,
+enum Shape {
+    Sphere { radius: f32 },
+    RectangularPrism { height: f32, width: f32, depth: f32 },
+}
+pub struct Shape {
+    shape: Shape,
+}
+
+pub struct Material {
```

Comment:
I'm not sure the `Material` name is a good idea considering Amethyst's renderer also uses a `Material` component for object textures. Maybe you could prefer something like `Content` which indicated what an entity could contain.

Reply:
true

Reply:
What about `MaterialComponent`?

Reply:
Not really, because 
1) Material is already a thing
2) We shouldn't append Component to components. (you could, but we don't internally)

Reply:
Having `Component` at the end of a component type is not good practice.

Reply:
Gotcha. Best part about working on the docs is you get to learn as you go. ;)

I changed it to `Content` as suggested. Truth be told, we probably just don't need multiple.